app/lib/data_exporter.rb

```diff
@@ -0,0 +1,42 @@
+require "yaml"
+
+class DataExporter
+  def self.call
+    new.call
+  end
+
+  def call
+    Rails.logger.info licence_data.each(&:deep_stringify_keys!).to_yaml
```

Comment:
It be nice to have this output tested but not a blocker as this will only be a temporary task. You've commented the results in a commit which feels like a good half way house 👍

Reply:
Yeah, I didn't see a need to test `to_yaml` when we are already testing the output from a nice readable hash.